src/SourceBuild/content/repo-projects/roslyn.proj

```diff
@@ -7,6 +7,7 @@
     <BuildActions Condition="'$(BuildOS)' == 'windows'">$(FlagParameterPrefix)restore</BuildActions>
     <BuildActions Condition="'$(BuildOS)' == 'windows'">$(BuildActions) $(FlagParameterPrefix)pack</BuildActions>
     <BuildActions Condition="'$(BuildOS)' == 'windows'">$(BuildActions) $(FlagParameterPrefix)publish</BuildActions>
+    <BuildActions Condition="'$(BuildOS)' == 'windows' and '$(DotNetBuildSourceOnly)' != 'true' and '$(Sign)' == 'true'">$(BuildActions) $(FlagParameterPrefix)sign</BuildActions>
```

Comment:
Why don't we sign roslyn on non-Windows?

Reply:
We do: `<!-- Roslyn's build.cmd adds -build automatically. On non-windows, use the default -->`

And from the ubuntu dev log: `/__w/1/vmr/src/roslyn/build.sh --restore --build --pack --publish --sign --ci --configuration Release ...`

Reply:
Perfect, thanks